src/main/webapp/VAADIN/themes/mytheme/layouts/medicationPrescription.html

```diff
@@ -0,0 +1,58 @@
+<div id="medDetailView">
+
+    <div id="medDetailViewTitle" data-location="medWindowTitle"></div>
+
+    <table id="medViewDetailTable">
```

Comment:
Refrain from using tables and instead use `div` and `label`

Reply:
Sprint 3. I din't want to spend the time. And tables are not deprecated at all. I know this is a point to discuss (if I were a web dev).